.release-please-manifest.json

```diff
@@ -15,5 +15,5 @@
     "packages/google-cloud-bigquery-biglake": "0.3.0",
     "packages/google-cloud-storageinsights": "0.1.0",
     "packages/google-cloud-confidentialcomputing": "0.1.0",
-    "packages/google-cloud-ids": "1.5.0"
+    "packages/google-cloud-ids": "1.0.0"
```

Comment:
Hmm, this isn't right. Why is it reverting?

Reply:
This has been fixed after following the steps in https://github.com/googleapis/release-please#how-do-i-change-the-version-number